phpunit/blocks/block-navigation-block-hooks-test.php

```diff
@@ -92,4 +112,29 @@ public function test_block_core_navigation_update_ignore_hooked_blocks_meta_pres
 			'Block was not added to ignored hooked blocks metadata.'
 		);
 	}
+
+	/**
+	 * @covers ::gutenberg_block_core_navigation_update_ignore_hooked_blocks_meta
+	 */
+	public function test_block_core_navigation_rest_creation() {
+		if ( ! function_exists( 'set_ignored_hooked_blocks_metadata' ) ) {
+			$this->markTestSkipped( 'Test skipped on WordPress versions that do not included required Block Hooks functionalit.' );
```

Comment:
```suggestion
			$this->markTestSkipped( 'Test skipped on WordPress versions that do not included required Block Hooks functionality.' );
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/59875/commits/668812d019f5f1ceb50fb14c44e835d58ad2da9c